filters.config.js

```diff
@@ -1,4 +1,38 @@
+const FILTER_VALUES = {
```

Comment:
suggestion: Move the FILTER_VALUES const into `aquarius/index.ts`

Reply:
I made the const into an enum and added it to the other methods in the searchQuery.ts is this ok?